augraphy/augmentations/delaunay.py

```diff
@@ -0,0 +1,230 @@
+
+        triangles = subdiv.getTriangleList()
+        triangles = triangles.astype(np.int32)
+        # adding perlin noise
+
+        if self.perlin:
+            obj_noise = Noise()
+            noise = np.array(
+                [
+                    [obj_noise.noise2D(j / 200, i / 200) * 50 + 200 for j in range(self.width)]
+                    for i in range(self.height)
+                ],
+                np.float32,
+            )
+            # noise = np.array((noise - np.min(noise)) / (np.max(noise) - np.min(noise)) * 255 , np.uint8)
+            nh, nw = noise.shape
+            # Convert the blue texture to grayscale
+            gray_texture = np.dot(noise[..., :3], [0.299, 0.587, 0.114])
+            white_texture = np.zeros((nh, nw, 3), dtype=np.uint8)
+            white_texture[..., 0] = gray_texture
+            white_texture[..., 1] = gray_texture
+            white_texture[..., 2] = gray_texture
+            img = cv2.addWeighted(
+                white_texture,
+                0.1,
+                img,
+                0.9,
+                0,
+            )  # creating a white texture from the perlin noise mesh
+            img = ndimage.gaussian_filter(img, sigma=(3, 3, 0), order=0)  # applying gaussian filter
+            colors = [
+                (240, 240, 240),
+                (250, 250, 250),
+                (234, 234, 234),
+                (244, 244, 244),
+                (254, 254, 254),
+                (243, 243, 243),
+            ]
+            img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
+            # Draw the Delaunay triangulation on the empty numpy array
+            for t in triangles:
+                pt1 = (t[0], t[1])
+                pt2 = (t[2], t[3])
+                pt3 = (t[4], t[5])
+                color = colors[np.random.randint(len(colors))]  # choose from colors
+                cv2.fillConvexPoly(img, np.array([pt1, pt2, pt3]), color)
+                cv2.line(img, pt1, pt2, (255, 255, 255), 1)
+                cv2.line(img, pt2, pt3, (255, 255, 255), 1)
+                cv2.line(img, pt3, pt1, (255, 255, 255), 1)
+        else:
+            min_val = self.color_background[0]
+            max_val = self.color_background[1]
+            for t in triangles:
+                pt1 = (t[0], t[1])
+                pt2 = (t[2], t[3])
+                pt3 = (t[4], t[5])
+                color = (
+                    np.random.randint(min_val, max_val),
+                    np.random.randint(min_val, max_val),
+                    np.random.randint(min_val, max_val),
+                )
+                cv2.fillConvexPoly(img, np.array([pt1, pt2, pt3]), color)
+        return img
+
+    # Applies the Augmentation to input data.
+    def __call__(self, image, layer=None, force=False):
+        if force or self.should_run():
+            result = image.copy()
+            h, w, _ = result.shape
+            delaunay_mesh = self.apply_augmentation()
+            # if self.perlin:
```

Comment:
Do we still need this section (line 221 to line 224)? Else we can just remove it. If you still need it, please include some related information, maybe something like `# Future development` or something else equivalent.